non-normative/recipes.adoc

```diff
@@ -30,3 +30,18 @@ configuration utilities. The actual meaning of the default (compatible to
 BRS-I) configuration would be highly specific to the vendor. In this example,
 it could capping all harts at the same performance level, or it could mean
 disabling efficiency harts.
+
+[[recipe-brs-m-guidance]]
+===== BRS-M Recipe Guidance
+
+In contrast to BRS-I and BRS-B, BRS-M has no implementation of UEFI related
+services. Non-UEFI firmware, such as *coreboot*, can also supports boot well.
+Although non-UEFI firmware does not support UEFI interface, it can also
+provide ACPI, SMBIOS and SBI services. ACPI and SMBIOS can be passed through
+DT.
+
+The introduction of coreboot can refer to https://www.coreboot.org/[coreboot].
+This solution is the startup mode for coreboot plus payload. coreboot does the
```

Comment:
This paragraph sounds more like a commit message than anything that would go into a standard.

It remains unclear why Coreboot is mentioned here. Is Coreboot a requirement of BRS-M?

If yes, say so. If not, define interfaces without relating to Coreboot.

Reply:
can we reduce line 37-47 to this?
BRS-M is intended to support software that has has reduced requirements for firmware, or that has alternative ways to access tables such as ACPI and SMBIOS. Some RISC-V-based systems, such as OpenTitan (see: opentitan.org), do not use tables such as ACPI or SMBIOS at all. In other cases, these tables are provided by new standards such as the Universal Payload Specification (https://github.com/UniversalPayload/spec). Finally, a kernel comtained in firmware might also have the tables comipiled-in or as part of its initial ram file system (initramfs in Linux). 

How does that sound? I'm trying to describe real world scenarios I have used.